### DISCUSSION TITLE
How to use refresh token in the package

### DISCUSSION THREAD

**Discussion by Kapoorrahul205** [spatie/dropbox-api#98] [Q&A]
I am looking to use Dropbox api using this package but package access toke expires after some time.I need to refresh token to avoid this but I did not get any specific documentation regarding this.

Is there any specific documentation to use refresh token with this package or how we can use refresh token with this package?

**Comment by freekmurze** [comment]
Refreshing the token is out of scope for this package. Check the Dropbox docs for more info.
